src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs

```diff
@@ -628,8 +628,8 @@ protected DbCommand GetCommand(DbConnection connection, string sql)
         protected virtual void WriteEvent(DbCommand command, IPersistentRepresentation e, IImmutableSet<string> tags)
         {
             var manifest = string.IsNullOrEmpty(e.Manifest) ? e.Payload.GetType().TypeQualifiedName() : e.Manifest;
-            var serializer = Serialization.FindSerializerFor(e.Payload);
-            var binary = serializer.ToBinary(e.Payload);
+            var serializer = Serialization.FindSerializerForType(typeof(IPersistentRepresentation));
```

Comment:
Existing users since 1.3.0? No one has been able to use Akka.Persistence on that version yet